R/manual_layer_delayed.R

```diff
@@ -2,8 +2,8 @@
 ##'
 ##' Note: this is a provisional API, which may be reworked before the next release
 ##' of this package.  In particular, there will be convenience wrappers like
-##' \code{delayedSQL} and \code{delayedArrayUDF}`, and the name and/or semantics of
-##' the base `delayed` function may soon change.
+##' \code{delayedSQL} and \code{delayedArrayUDF}, and the name and/or semantics of
+##' the base \code{delayed} function may soon change.
```

Comment:
I actually catch myself doing that too, and I have written .Rd files for so many moons.

You can in fact turn on (R)Markdown (pre-)processing for roxygen2 via `Roxygen: list(markdown = TRUE)` in `DESCRIPTION but I personally prefer my setup simpler.